src/Engine/ProtoCore/Lang/BuiltInMethods.cs

```diff
@@ -60,6 +60,7 @@ public enum MethodID
             Transpose,
             Union,
             InlineConditional,
+            ConditionalIf,
```

Comment:
because this enum is public - to avoid reordering the values in the list- this new enum should added at the end of the list, or assigned an int value greater than the last value in the list.

ie 
`ConditionalIf = 1000`

Reply:
If you put this one in the middle of the list, even if you set the value, you will still need to explicitly set the following one as well to keep its current value.
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/enums

Reply:
I will add it to the end of the list which seems safer.